duxbay.conf

```diff
@@ -32,6 +32,11 @@ LIPATH=${LUSER}/ingest
 
 SPK_EXEC='400'
 SPK_EXEC_MEM='2048m'
+SPK_DRIVER_MEM=''
```

Comment:
so, is the idea that these correspond to unspecified or default values?

or does the user *have* to update these values in duxbay.conf during installation?